sal_interfaces/PMD/PMD_Events.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-stylesheet type="text/xsl" href="http://lsst-sal.tuc.noao.edu/schema/SALEventSet.xsl"?>
+<SALEventSet xmlns:xsi="http://lsst-sal.tuc.noao.edu/schema/SALEventSet.xsd">
+  <SALEvent>
+    <Subsystem>PMD</Subsystem>
+    <EFDB_Topic>PMD_logevent_metadata</EFDB_Topic>
+    <item>
+      <EFDB_Name>kind</EFDB_Name>
+      <Description>The kind of measurement i.e.micrometers or incremental rotary encoders</Description>
+      <IDL_Type>string</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>location</EFDB_Name>
+      <Description>Overall location of the device.</Description>
+      <IDL_Type>string</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>names</EFDB_Name>
+      <Description>Comma-separated list of names that details what each device measures.</Description>
+      <IDL_Type>string</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+    </item>
+    <item>
+      <EFDB_Name>units</EFDB_Name>
+      <Description>The unit of the devices.</Description>
```

Comment:
you mean the units of the values published by the device?
Does this have to do with the comma separated list above? I suppose we're assuming that each device cannot publish two columns of information in different units?

Reply:
It is the units of the reported measurements. One value for all measuring devices (for a given instance/SAL index of PMD). For example we will presumably set our Mitutoyo micrometers to report measurements in um, in which case this string would be "um".

This assumes we don't have some weird hybrid system where some devices attached to a box measure in um and some in degrees or.... I sure hope we can assume that. For now all we have is Mitutoyo micrometers.